src/TransactionalMigrationInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Db\Migration;
+
+/**
+ * Allowing to use transaction in migration.
+ */
+interface TransactionalMigrationInterface
```

Comment:
done.

Reply:
In this case i am not very sure, because if i wanted to create a migration with a reversible transaction, i must do it manually, i think that the interface transaction should extend the two interfaces.

Reply:
Command `migrate/create` will be create `RevertibleMigrationInterface, TransactionalMigrationInterface`, but user will have the opportunity to manually create `TransactionalMigrationInterface`.